NetTopologySuite/Operation/Union/UnionInteracting.cs

```diff
@@ -44,7 +44,7 @@ public IGeometry Union()
             // check for all interacting or none interacting!
             IGeometry int0 = ExtractElements(_g0, _interacts0, true);
             IGeometry int1 = ExtractElements(_g1, _interacts1, true);
-#if !PCL
+#if DEBUG

```

Comment:
I'm not interested in defending this change.  JTS calls `isEmpty()` no matter what.  NTS can do the same.